### ISSUE TITLE
Migrating away from qTranslate

### ISSUE THREAD

**Issue by anttius** [qTranslate-Team/qtranslate-x#615]
Hey.

Our companies website was build with divi builder together with qtranslate. They are not compatible and its causing a lot of problems. Apparently there are no working plugins to migrate to polylang from qTranslate.

I was thinking of saving every single page and project as a divi layout template - removing qTranslate with one language left and building new language sites from the layouts with polylang.

Problem is when i deactivate qTranslate, there is no one default language left while the other disappears, but there is a mess of 3 languages combined.

Is there a way to get through this somehow? 

Thanks all!

- Antti

**Comment by herrvigg** [comment]
qTranslate-X is a dead repo for 3 years so you won't get help here. We have been reviving it with qTranslate-XT and it seems some people are using with Divi (e.g. https://github.com/qtranslate/qtranslate-xt/issues/717) althrough there's no specific change to handle this plugin so far so i don't see why it wouldn't work with qTranslate-X, there should be something else explaining this.

If you migrate to another tool you have to deal with the raw values by yourself.

**Comment by Kipperlenny** [comment]
It's not working with DIVI.
But I read, you can use qtranslate to WPML migration plugin (you don't need wpml for that) and afterwards you can use the WPML to Polylang Migration Plugin.

**Comment by camfil26** [comment]
No, you will be able to import only one language without the WPML standard plugin
